open-scd.test.ts

```diff
@@ -0,0 +1,333 @@
+import { visualDiff } from '@web/test-runner-visual-regression';
+
+import type { IconButton } from '@material/mwc-icon-button';
+import type { ListItem } from '@material/mwc-list/mwc-list-item.js';
+
+import './open-scd.js';
+import { expect } from '@open-wc/testing';
+import type { OpenSCD } from './open-scd.js';
+
+import { Edit, newEditEvent, newOpenEvent } from './foundation.js';
+import { allLocales } from './locales.js';
+
+const factor = window.process && process.env.CI ? 4 : 2;
+
+function timeout(ms: number) {
+  return new Promise(res => setTimeout(res, ms * factor));
+}
+
+mocha.timeout(2000 * factor);
+
+const doc = new DOMParser().parseFromString(
+  `<testdoc></testdoc>`,
+  'application/xml'
+);
+
+let editor: OpenSCD;
+beforeEach(() => {
+  editor = document.createElement('open-scd');
+  document.body.prepend(editor);
+});
+
+afterEach(() => {
+  editor.remove();
+});
+
+it(`changes locales on attribute change`, async () => {
```

Comment:
This is a test we already have in `visual.test.ts`. Is this duplication on purpose?

Reply:
Ahhh now I get it. You wanted to rename `visual.test.ts` to `open-scd.test.ts`. I think this is why the screenshots fail. You are creating new screenshots, but compare to the old once. Also, please remove the leftover `visual.test.ts`

Reply:
Where is the leftover `visual.test.ts`? I can't find it for some reason.

Reply:
Looks like this was completely unrelated to the file rename and just had to do with lit-localize's `locales/de.js` not being picked up in the distribution directory. After moving back from `inputFiles` to `tsConfig` in `lit-localize.json`, localizations are now being emitted as typescript again (`locales-de.ts`) and therefore being included in `dist` by `tsc`, resolving the issue.

Reply:
What it says under https://lit.dev/docs/localization/cli-and-config/#general-settings is not true by the way, `inputFiles` does not supercede `tsConfig`, it's sadly the other way round. So you can't currently have custom input files while using TypeScript.